pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -338,5 +381,8 @@ func (f *Fit) Score(ctx context.Context, state *framework.CycleState, pod *v1.Po
 		return 0, framework.AsStatus(fmt.Errorf("getting node %q from Snapshot: %w", nodeName, err))
 	}
 
+	s, _ := getPreScoreState(state)
```

Comment:
For the better readability, instead of resource_allocation.go#L88, if no PreScore state is found in `state`, we should just re-calculate the same thing here.